### ISSUE TITLE
Harvesters spawn in large numbers in Mushroom Island biomes

### ISSUE THREAD

**Issue by dozingpip** [Lykrast/HarvestersNight#6]
Since it seems to be the only hostile mob (in my custom pack at least) able to spawn on the island, I think there's a much higher chance of them spawning.  The first night in the world (I picked a seed that had a mushroom island close to spawn) 4 of them spawned very close together time-wise and I was unable to kill any of them since I only had a pick for a weapon...  For now I might turn off the mod entirely for playing this particular world.  I'm open to any other potential suggestions though, like maybe I'm missing something in the config that prevents them spawning so frequently in this biome.

**Comment by Lykrast** [comment]
> Since it seems to be the only hostile mob (in my custom pack at least) able to spawn on the island

Exactly, I need to see if I can put a biome black/whitelist on top of the dimension one. I think you can use In Control to manually change spawns if needed.
